### DISCUSSION TITLE
Make alarm notifications and service notifications available as triggers in HA

### DISCUSSION THREAD

**Discussion by Fyru** [SukramJ/aiohomematic#1228] [Ideas]
Would it be possible to introduce a HA trigger for the presence of notifications? Otherwise, I don't see a good way to know when there are updates without opening the web ui. I would also appreciate any suggestions for alternative ways to implement this.

**Comment by SukramJ** [comment]
It makes sense to choose a language. Preferably your native one

**Reply by Fyru** [reply]
Okay, I changed it to English.

**Comment by danielperna84** [comment]
Do you mean the "Servicemeldungen"? There should be a variable-entity like _sensor.ccublabla_sv_servicemeldungen_, which has the number of current messages. You can use that in an automation if the value changes above 0.

**Reply by Fyru** [reply]
Hey, thanks for your response. Unfortunately, I can't find anything similar. Where exactly should I look? I've searched for a corresponding entity in the developer tools.

**Comment by emufan** [comment]
There should be a ccu device and numerous of hidden sensors.

![image](https://github.com/danielperna84/hahomematic/assets/22775515/6862e48a-cdb0-42a0-8a0f-89449e89927b)

**Comment by Fyru** [comment]
Oh, okay. Thanks for the screenshot. I think I'm in the wrong GitHub repository. Sorry for that.
![image](https://github.com/danielperna84/hahomematic/assets/1583273/b37e9113-df1c-4eb9-ac49-a5e68312ac44)

**Comment by Fyru** [comment]
Or maybe not. I can't really determine if the issue is here or with RaspberryMatic.

**Reply by SukramJ** [reply]
Have you read the [docs](https://github.com/danielperna84/custom_homematic#deactivated-entities)?

**Reply by emufan** [reply]
Klick auf das Zahnrad und aktiviere die Entität. Wie von mir oben ja schon beschrieben "hidden".

**Reply by Fyru** [reply]
Thank you, that really helped me. It's working now.
